src/lib/Key.js

```diff
@@ -91,6 +161,124 @@ class Key {
             : this._secret;
     }
 
+    /**
+     * @param {EncryptionKeyParams} keyParams
+     * @returns {Promise<CryptoKey>}
+     */
+    async getRsaPrivateKey(keyParams) {
+        if (!this.rsaKeyPair || !Key._isEncryptionKeyParamsEqual(keyParams, this.rsaKeyPair.keyParams)) {
+            this.rsaKeyPair = await this._computeRsaKeyPair(keyParams);
+            if (Key._isDefaultEncryptionKeyParams(keyParams)) {
+                await KeyStore.instance.setRsaKeypair(this.id, this.rsaKeyPair);
+            }
+        }
+
+        return window.crypto.subtle.importKey(
+            'pkcs8',
+            this.rsaKeyPair.privateKey,
+            { name: 'RSA-OAEP', hash: 'SHA-256' },
+            false, // Prevent extraction
+            ['decrypt'],
+        );
+    }
+
+    /**
+     * @param {EncryptionKeyParams} keyParams
+     * @returns {Promise<CryptoKey>}
+     */
+    async getRsaPublicKey(keyParams) {
+        if (!this.rsaKeyPair || !Key._isEncryptionKeyParamsEqual(keyParams, this.rsaKeyPair.keyParams)) {
+            this.rsaKeyPair = await this._computeRsaKeyPair(keyParams);
+            if (Key._isDefaultEncryptionKeyParams(keyParams)) {
+                await KeyStore.instance.setRsaKeypair(this.id, this.rsaKeyPair);
+            }
+        }
+
+        return window.crypto.subtle.importKey(
+            'spki',
+            this.rsaKeyPair.publicKey,
+            { name: 'RSA-OAEP', hash: 'SHA-256' },
+            true, // Allow extraction
+            ['encrypt'],
+        );
+    }
+
+    /**
+     * @param {EncryptionKeyParams} keyParams
+     * @returns {Promise<RsaKeyPairExport>}
+     */
+    async _computeRsaKeyPair(keyParams) {
+        const iframe = document.createElement('iframe');
```

Comment:
Would be good to add a comment that explains why we're doing all this fun with the iframe, i.e. browser APIs don't support the deterministic generation of RSA keys from a seed, therefore we use a library, but to avoid an increased attack surface by including a library not audited by us, we run it in a sandbox.
```suggestion
        // Setup the RSAKeysIframe in which the actual computation of the RSA key happens via node-forge. The reason why
        // we don't use the browser crypto API instead, is that it does not support deterministic generation of RSA keys
        // from an existing secret, and the reason why node-forge is not used directly here is that we want to run it in
        // a sandbox, to reduce the attack surface from a big external dependency, not thoroughly audited by us.
        const iframe = document.createElement('iframe');
```

Reply:
Resolved in #524